packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_fr_FR.json

```diff
@@ -115,5 +115,8 @@
   "sms_sms_compose_add_phonebook_contact_empty": "Aucun contact trouvé.",
   "sms_sms_compose_add_phonebook_contact_adding": "Mise à jour des contacts…",
   "sms_sms_compose_add_phonebook_contact_added": "Liste des contacts mise à jour !",
-  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts."
-}
\ No newline at end of file
+  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts.",
+  "sms_sms_compose_message_commercial_description1": "Pour des raisons légales, un SMS à caractère commercial ne sera envoyé qu’",
+  "sms_sms_compose_message_commercial_description2": "entre 8 h et 20 h, du lundi au samedi.",
```

Comment:
You didn't answer last time 😅 . Are those value valid in Spain too ?

Reply:
I answered ;-) Shortcode sender isn't use in Spain.

Reply:
Sorry, did't see that, good point I guess :)